lib/hexpm_web/controllers/api/package_controller.ex

```diff
@@ -12,8 +12,13 @@ defmodule HexpmWeb.API.PackageController do
     repositories = repositories(conn)
     page = Hexpm.Utils.safe_int(params["page"])
     search = Hexpm.Utils.parse_search(params["search"])
-    sort = sort(params["sort"])
-    packages = Packages.search_with_versions(repositories, page, 100, search, sort)
+
+    sort_by_field = sort(params["sort"])
+    if params["sort"] == nil and search != nil and search =~ "updated_after:" do
+      sort_by_field = nil
+    end
```

Comment:
If this isn't used, the query contains `ORDER BY NAME DESC, UPDATED_AT ASC` which makes the pagination ineffective. 

Feel free to suggest a better approach :)

Reply:
Instead of doing this you can use `Ecto.Query.exclude(query, :order_by)` in `search_param("updated_after", ...)`.

Reply:
Nice. I didn't know we can do that. Thanks!